src/content/docs/tutorial-diagnose-infra-bottlenecks/instrument-your-system.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Collect data about your system infrastructure
+metaDescription: Capture data about your system by installing agents, then view that data in New Relic. 
+---
+
+Like the bridges and roads of a dense city, your system infrastructure needs regular inspection and maintenance. What processes affect CPU and disk usage? What app behaviors typically correlate with failing hosts? The performance of your infrastructure can affect the performance of your APIs, services, and apps. A database failure can prevent your server from handling data, or a memory leak can bring an otherwise performant app to an abrupt halt. 
+
+## Objectives [#objectives]
+
+This tutorial series walks you through using infrastructure data to troubleshoot different incident scenarios. While New Relic offers a full suite integrations to monitor the Cloud, containerized environments, and various virtual machines, this series is specific to data reported with an infrastructure agent that does not have additional integrations.
```

Comment:
a full suite _of_ integrations or: a full suite integration

Reply:
Can you explain the difference? Like is this the "infrastructure fundamentals" and everything else is just icing on that cake?

Reply:
I would say it's more that sure, you can get by with just the infra install, but if you want to actually solve problems effectively you should really collect as many data points as possible, right.